ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -231,11 +231,6 @@ QtObject {
         stickersModuleInst.send(channelId, hash, replyTo, pack, url)
     }
 
-    // TODO: This seems to be better in Utils.qml
-    function copyToClipboard(text) {
```

Comment:
There seems to be a lot more occurrences of `store.copyToClipboard` (~50 more matches)

Reply:
![image](https://github.com/status-im/status-desktop/assets/5377645/7eabf59c-e21d-4349-8ec1-43b724824f97)

I simply wouldn't this kind of (unrelated) refactoring here

Reply:
yeah sure there are, I just took care of chat section here (only where the pr is changing files), we could add a task for the rest, wdyt

Reply:
No, I mean you just removed it from the chat's RootStore but I'm almost sure there are still places using this

Reply:
yeah, under chat section is replaced wherever it was used (see https://github.com/status-im/status-desktop/pull/12090/files#diff-2d873fc1cf743182a33afddcfe6e1214586f803b7e3a0eae4ac1145ebc566951)